dsgrid/utils/dataset.py

```diff
@@ -108,6 +108,48 @@ def map_and_reduce_pivoted_dimension(df, records, pivoted_columns, operation, re
     return df.selectExpr(*nonvalue_cols, *extra_cols, *exprs), sorted(final_columns), dropped
 
 
+def make_time_zone_mapping(load_data_df, geography_dim):
```

Comment:
@lixiliu This function is a modification of older code and is not used. I'm inclined to delete it, and so let me know if you want to keep it.

Reply:
sure go for it